index.html

```diff
@@ -218,6 +237,400 @@ <h2>
+        </aside>
+        <p>
+          An application <dfn>has a home tab</dfn> if the [=Document/processed
+          manifest=] includes a non-null [=home_tab=] member of the
+          [=tab_strip=] member.
+        </p>
+        <p>
+          The <dfn>home tab context</dfn> is an optional [=application
+          context=] that has special properties compared to other application
+          contexts. If the application [=has a home tab=], every application
+          window SHOULD feature a [=home tab context=]. If not, then the
+          application windows SHOULD NOT have a [=home tab context=].
+        </p>
+        <p>
+          How the [=home tab context=] is presented is at the discretion of the
+          user agent, but it SHOULD have a different appearance to normal tabs.
+        </p>
+        <p>
+          A [=URL=] |url:URL| is said to be <dfn>within home tab scope</dfn> if
+          and only if:
+        </p>
+        <ul>
+          <li>the application [=has a home tab=], and
+          </li>
+          <li>|url| is [=manifest/within scope=] of the manifest, and
+          </li>
+          <li>at least one of the following:
+            <ul>
+              <li>|url| [=URL/equals=] the [=start URL=], with
+              [=URL/equals/exclude fragments=] set to true, or
+              </li>
+              <li>Applying [=URL pattern/match=] given any element of the
+              [=scope_patterns=] member of the [=home_tab=] member of the
+              [=tab_strip=] member of the [=Document/processed manifest=] and
+              |url| returns a {{URLPatternResult}}.
+              </li>
+            </ul>
+          </li>
+        </ul>
+        <p>
+          A URL is is <dfn>outside of home tab scope</dfn> if it is not
+          [=within home tab scope=].
+        </p>
+        <aside class="note">
+          <p>
+            The "home tab scope" (i.e., the set of [=URLs=] which are [=within
+            home tab scope=]) is the set of URLs to which the [=home tab
+            context=] can be navigated . All navigations within the application
+            to a URL [=within home tab scope=] will be performed in the [=home
+            tab context=], and all navigations to a URL [=outside of home tab
+            scope=] will be performed in a different [=application context=]
+            (i.e., another tab), creating one if necessary. The application's
+            [=start URL=] is always part of the home tab scope.
+          </p>
+          <p>
+            For the purpose of matching against the [=start URL=], the
+            [=URL/fragment=] is ignored, but the [=URL/query=] must match
+            exactly.
+          </p>
+          <p>
+            If the application does not [=has a home tab|have a home tab=],
+            then the home tab scope is the empty set.
+          </p>
+        </aside>
+        <section>
+          <h3>
+            Every window has a home tab
```

Comment:
There are some assumptions in this text that the browsing contxt is in `tabbed` display mode. The definition of the `application context` doesn't differentiate windowed vs in the browser. It might be good to clarify that the context for some of these assertions is that the window is in tabbed display mode.... you may need to define a mulit-browser-context window? Not sure if that is in spec somewhere.

Reply:
Louise just raised this: I have amended the definition of "has a home tab" to require that the display mode is tabbed, which I think addresses this.

> The definition of the application context doesn't differentiate windowed vs in the browser.

I'm pretty sure it does (though not very clearly) - the definition of "[application context](https://www.w3.org/TR/appmanifest/#dfn-application-context)" is "A top-level browsing context that has a manifest applied to it" which my understanding means an application window. Otherwise I think a lot of our assumptions throughout the Manifest spec are flawed.